### PR TITLE
Added support for gentoo (Local deployment only)

### PR DESCRIPTION
**Pull request by siddhanthrathod**

### CODE REVIEW

**Review by rodrigosiqueira — CHANGES_REQUESTED** [review]
In lgtm, but I want to see tests and remote deploy support.

File: documentation/dependencies/gentoo.dependencies
[the remark this answers is not in the document]
Comment (siddhanthrathod) [reply documentation/dependencies/gentoo.dependencies:1]: Okay

File: src/plugins/kernel_install/gentoo.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    prefix="${path_prefix}"
+  fi
+
+  # We do not support initramfs outside grub scope
+  [[ "$bootloader_type" != 'GRUB' ]] && return
+
+  cmd+=" $name"
+
+  if [[ "$target" == 'local' ]]; then
+    cmd="sudo -E $cmd"
```
[the remark this answers is not in the document]
Comment (siddhanthrathod) [reply src/plugins/kernel_install/gentoo.sh:53]: sorry will do

**Comment by siddhanthrathod** [comment]
Okay I will do that

**Comment by rodrigosiqueira** [comment]
Any update here?

**Comment by rodrigosiqueira** [comment]
I'm closing this PR. If you have a new version, please re-open it.

**Pull request closed without merging**